loadouts/content/broma-weapon-list.sqf

```diff
@@ -526,11 +555,30 @@ _HLC416C = ["hlc_rifle_416C",_30rnd556AR15];
 _HLC416N = ["hlc_rifle_416N",_30rnd556AR15];
 _HLC416NGL = ["hlc_rifle_416N_gl",_30rnd556AR15,_40mm];
 
-//========== HLC M16 PACK =======================================================
```

Comment:
Why was this deleted and the weapons moved to `AR PACK`?

Reply:
oh, they were part of the HLC AR pack, so i decided t oassamble it all together so if someone is looking for their classnames he can found them here ( was there really an M16 pack or was it just the AR pack that got updated ? )

Reply:
just checked, they are all part of the AR-15 pack

Reply:
Ah, I think I understand what happened. I assume Royal probably added this section based on `@NIArms All in One` PBO's which has `hlc_wp_M16A2.pbo`. However, there isn't actually an M16 pack and that PBO and `hlc_wp_AR15.pbo` is apart of `@NIArms AR15 Rifles`.